shared/api/auth.go

```diff
@@ -105,6 +105,41 @@ type IdentityPut struct {
 	Groups []string `json:"groups" yaml:"groups"`
 }
 
+// IdentitiesPostTLS contains required information for the creation of a TLS identity.
+//
+// swagger:model
+//
+// API extension: access_management_tls.
+type IdentitiesPostTLS struct {
```

Comment:
I believe the verb goes at the end, so this should be IdentitiesTLSPost right?

Reply:
Done.